### ISSUE TITLE
Error attempting to merge large files.

### ISSUE THREAD

**Issue by garnertb** [mapbox/geojson-merge#6]
```
>>> ls -la -h addresses_area.geojson 
-rw-r--r--  1 s30226  staff   272M Aug 27 19:20 addresses_area.geojson

>>> ogrinfo addresses_area.geojson 
ERROR 4: GeoJSON Driver doesn't support update.
Had to open data source read-only.
INFO: Open of `addresses_area.geojson'
      using driver `GeoJSON' successful.
1: OGRGeoJSON

>>> geojson-merge addresses_area.geojson 
buffer.js:488
    throw new Error('toString failed');
    ^

Error: toString failed
    at Buffer.toString (buffer.js:488:11)
    at Object.parse (native)
    at /usr/local/lib/node_modules/geojson-merge/geojson-merge:6:17
    at Array.map (native)
    at Object.<anonymous> (/usr/local/lib/node_modules/geojson-merge/geojson-merge:5:65)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)

```

toString failed seems to be coming from this: https://github.com/nodejs/node/issues/3175.

Any ideas how I can work around this?

**Comment by drewbo** [comment]
@garnertb I used something like this to concat large GeoJSON:

https://gist.github.com/drewbo/e6fcc6958571e2fb6b3823c7aafcefd2

@tmcw any desire to see this added or worth abstracting as another module?

**Comment by tmcw** [comment]
💭  That's a clever approach @drewbo! My only reservation related to replacing the logic of this module _completely_ is that we currently use geojson-normalize to allow people to merge a Feature and a Feature, or a FeatureCollection and a Geometry, etc. I always want to rally for 100% "all the things GeoJSON can do" compatibility rather than assuming FeatureCollections at the top level.

geojson-stream doesn't normalize and probably can't, without adding a lot of complexity (or, idk, there's probably a way that streaming-data-pros know and I don't).

Maybe we should add big-files mode as an option to the CLI so that people can get flexibility on small data and performance on large?

**Comment by drewbo** [comment]
Very good point. I'll like the idea of a `--big` flag which only accepts `FeaturesCollections` and letting the base module retain the flexibility that comes with normalization. I'll push a small PR in a few to get things started

**Comment by tmcw** [comment]
Fixed in 409bff784e5419ebf0126b1722f1f73bf90fa223
